lib/tbot/readyz/status.go

```diff
@@ -0,0 +1,83 @@
+/*
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package readyz
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+// Status describes the healthiness of a service or tbot overall.
+type Status uint
+
+const (
```

Comment:
Do we think a "starting"/"initializing" state might be useful? For example, whilst they are waiting for another dependency to come online?

Reply:
Yeah that might be nice - more of a passing thought than anything so if Unknown makes sense for other reasons then that's fine.